client/src/docs-ui/hero/hero.tsx

```diff
@@ -0,0 +1,29 @@
+import {Component, h, Host, State, Listen} from "@stencil/core";
+import {heroStyle, landingSearchStyle} from "./hero.style";
+import {isWiderThanTablet} from "../../amplify-ui/styles/media";
+
+@Component({tag: "docs-hero", shadow: false})
+export class DocsHero {
+  @State() isWiderThanTablet = this.calculateIsWiderThanTablet();
+
+  @Listen("resize", {target: "window"})
+  calculateIsWiderThanTablet() {
+    this.isWiderThanTablet = isWiderThanTablet();
```

Comment:
Nit: Why not name this expressively for its purpose? In this case, it's `showSearch` or something similar. On L21, it's hard to even see the bang, so it might benefit from more purposeful naming.